python/wpt/run.py

```diff
@@ -99,7 +99,7 @@ def run_tests(default_binary_path: str, **kwargs):
     wptcommandline.check_args(kwargs)
 
     mozlog.commandline.log_formatters["servo"] = (
-        ServoFormatter,
+        tuple[type[ServoFormatter], str],
```

Comment:
I'm not sure I understand this expression here - the syntax looks like a type, but I guess we need a runtime value here. How does this work?

Reply:
The type checking works, but `./mach test-wpt` (i.e running the affected method) doesn't work though:
```
TypeError: type() takes 1 or 3 arguments
```
Which makes sense because the value of `log_formatters["servo"]` must be a tuple whose first component is the class itself. We probably need to have an explicit [cast](https://mypy.readthedocs.io/en/stable/type_narrowing.html#casts) here using `cast(Callable, ServoFormatter)`